installers/ansible/values.yaml

```diff
@@ -20,7 +20,7 @@ badger: "false"
 ccp_image_prefix: "perconalab/percona-postgresql-operator"
 ccp_image_pull_secret: ""
 ccp_image_pull_secret_manifest: ""
-ccp_image_tag: "main-ppg14-postgres-ha"
+ccp_image_tag: "main-ppg15-postgres-ha"
```

Comment:
ppg14 is staying here, since we will not support PG 15 in this release, but we need to see what was here on the previous release